src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -342,6 +342,22 @@ public void addStudentRespondent(String email, FeedbackSessionAttributes feedbac
         addStudentRespondents(emails, feedbackSession);
     }
 
+    public Instant softDeleteFeedbackSession(FeedbackSessionAttributes feedbackSession)
```

Comment:
Javadoc + testing :P

Reply:
The parameter can just be `String feedbackSessionName, String courseId`.

The problem of passing a DTO to the method in DB for updating is that you don't know whether the fields inside the DTS is changed or not.